templates/kubernetes-cluster.template

```diff
@@ -949,6 +949,16 @@ Resources:
         InstancePort: 6443
         InstanceProtocol: TCP
         LoadBalancerPort: 443
+      - Protocol: TCP
```

Comment:
hmm what's this for? I'd expect we still only need the 443 => 6443 listener

Reply:
The reasoning behind this is that kubeadm api port is 6443, but the ELB listens on port 443 only. On init, kubeadm sets up apiserver liveness probe to check on port 6443 on the ELB's ip, which isn't available. We could either change the api port to be 443 and match the ELB one, or (as above) add a second listener.

Reply:
I think this is a result of overriding the advertise url.

Reply:
Do we still need this port on the LB now?